cni/network/invoker_mock.go

```diff
@@ -79,18 +79,20 @@ func (invoker *MockIpamInvoker) Add(opt IPAMAddConfig) (ipamAddResult IPAMAddRes
 	return ipamAddResult, nil
 }
 
-func (invoker *MockIpamInvoker) Delete(address *net.IPNet, nwCfg *cni.NetworkConfig, _ *skel.CmdArgs, options map[string]interface{}) error {
+func (invoker *MockIpamInvoker) Delete(addresses []*net.IPNet, nwCfg *cni.NetworkConfig, _ *skel.CmdArgs, options map[string]interface{}) error {
 	if invoker.v4Fail || invoker.v6Fail {
 		return errDeleteIpam
 	}
 
-	if address == nil || invoker.ipMap == nil {
+	if len(addresses) <= 0 || invoker.ipMap == nil {
```

Comment:
Is there a case where  length can be < 0 ?

Reply:
Change this to len(addresses) == 0